NEWS.md

```diff
@@ -7,6 +7,11 @@
 * DESCRIPTION file includes URL and BugReports sections (#125).
 * Made a few syntax changes in vignettes (#101).
 * Changed Pillars nomenclature to be consistent with official terminology (#103).
+* Set parameters initialized in global, Annual Contribution initialized at 5000, 
```

Comment:
I would not include the tech. details about the intialization in global, what matters more is the initial value of P3 and forcing negative values to 0 and whether we have removed some upper bound check.

Updating roxygen and commit-like `fixes #...` also do not belong here